zsh/functions/tat

```diff
@@ -30,8 +30,9 @@ create_if_needed_and_attach() {
 }
 
 tat() {
-  path_name="$(basename "$PWD" | tr . -)"
-  session_name=${1-$path_name}
+  parent="$(basename "$(dirname "$PWD")")"
```

Comment:
Fun fact: You don't need to quote the right side of assignment. So this could be:

```sh
parent=$(basename "$(dirname "$PWD")")
```